virttest/libvirt_xml/domcapability_xml.py

```diff
@@ -78,9 +79,11 @@ def get_additional_feature_list(self, cpu_mode_name):
                     for feature in mode_node.findall('feature'):
                         item = {}
                         item[feature.get('name')] = feature.get('policy')
-                        # Feature invtsc doesn't support migration, so not add it to feature list
-                        if 'invtsc' not in item:
-                            feature_list.append(item)
+                        if ignore_invtsc and 'invtsc' in item:
```

Comment:
Is there any chance we ignore other features too? If there is, are we considering adding an unsupported list instead of ignoring feature one by one ?

Reply:
Thank you @chloerh for reviewing it. I just found the behavior of this case has been changed, not sure what the problem was.
So I will convert it soon for further investigation.

Reply:
Weird, the problem is gone on the latest version. Anyway, updated.